gpkit/feasibility.py

```diff
@@ -1,8 +1,7 @@
 import numpy as np
-
-from .variables import Variable, VectorVariable
+from .nomials import NomialArray
+from .nomials import Variable, VectorVariable
 from .varkey import VarKey
-from .posyarray import PosyArray
 
 
 def feasibility_model(program, flavour="max", varname=None,
```

Comment:
right below this line you'll see more arguments -- should `signomials` be renamed?